apache/trafficcontrol
# Do not depend on Traffic Ops static web server

## What does this PR (Pull Request) do?

- - [x] This PR fixes #5683 by no longer attempting to download MaxMind GeoIP databases during Postinstall (813c4e7619)
- - [x] This PR fixes #5587 by
    - Removing references to Perl-only parameters that depended on `/opt/traffic_ops/app/public`:
        - `geniso.iso_root_path` from `cdn.conf` for downloading ISOs after the original `POST /isos` response that includes it (ec62b70071)
        - The `tm.logourl` profile parameter, used for specifying a URL to a Traffic Ops logo for use in the Traffic Ops UI (23fb5ffa02)
    - Hosting the MaxMind GeoIP database and coverage zone file on a static web server in CDN in a Box (9ab82f7430)

## Which Traffic Control components are affected by this PR?

- CDN in a Box
- Documentation
- Traffic Ops
- GitHub Actions

## What is the best way to verify this PR?

- Verify that the Traffic Ops RPM no longer creates the `/opt/traffic_ops/app/public` directory

- Run CDN in a Box, ensure that, for Traffic Router:
  - `MaxmindGeolocationService` creates `/opt/traffic_router/db/GeoLite2-City.mmdb`
  - `NetworkUpdater` creates `/opt/traffic_router/db/czmap.json`

## If this is a bug fix, what versions of Traffic Control are affected?

- master (da871ea812)

## The following criteria are ALL met by this PR
